dapp/pages/stake.js

```diff
@@ -200,7 +201,19 @@ const Stake = ({ locale, onLocale, rpcProvider, isMobile }) => {
         }
         tokenToStakeDecimalsCall={ognContract.decimals}
         stakeFunctionCall={async (stakeAmount) => {
-          return ognStaking.stake(stakeAmount, selectedDuration)
+          //const stakeAmountString = formatBn(stakeAmount, 18)
+          const interfce = ognStaking.interface
```

Comment:
nit: typo "interface"

Reply:
Renaming to `iface` so it looks like less of a typo, couldn't use `interface` since it is a reserved keyword 